x/cdp/spec/06_params.md

```diff
@@ -8,12 +8,16 @@ The cdp module contains the following parameters:
 | DebtParams                   | DebtParam               | {see below}                        | array of params for each enabled pegged asset                    |
 | GlobalDebtLimit              | coin                    | {"denom":"usdx","amount":"1000"}   | maximum pegged assets that can be minted across the whole system |
 | SavingsDistributionFrequency | string (int)            | "84600"                            | number of seconds between distribution of the savings rate       |
-| CircuitBreaker               | bool                    | false                              | flag to disable user interactions with the system                |
+| GlobalDebtLimit              | coin                    | {"denom":"usdx","amount":"1000"}   | maximum pegged assets that can be minted across the whole system |
```

Comment:
Not sure how Kava renders its website documentation, but you may have to escape (via backticks) any `{}`.

Reply:
Ty for the heads-up, this is indeed causing issues. Will make sure it gets addressed in separate PR.